app/code/Magento/Shipping/Block/Items.php

```diff
@@ -24,6 +24,13 @@ class Items extends \Magento\Sales\Block\Items\AbstractItems
      */
     protected $_coreRegistry = null;
 
+    /**
+     * Zend Validator Uri
+     *
+     * @var \Zend\Validator\Uri
+     */
+    protected $_zendValidatorUri;
```

Comment:
This property, together with it's assignment does not seem to have any clients. This may be an error.